frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -241,9 +245,61 @@ function useCreateHmpbTargetElements({
 const BmdPaperBallotsMemoized = React.memo(BmdPaperBallots);
 const HandMarkedPaperBallotsMemoized = React.memo(HandMarkedPaperBallots);
 
+interface PrintingModalProps {
+  advancePrinting: () => void;
+  currentPrecinct: Precinct;
+  precinctIds: string[];
+  printIndex: PrintIndex;
+}
+
+function PrintingModal({
+  advancePrinting,
+  currentPrecinct,
+  precinctIds,
+  printIndex,
+}: PrintingModalProps): JSX.Element {
+  if (printIndex.component === 'PaperChangeModal') {
+    return (
+      <Modal
+        centerContent
+        content={
+          <Prose textCenter>
+            <p>
+              Load printer with <b>legal-size paper</b>.
+            </p>
+          </Prose>
```

Comment:
Let's use a heading to add some hierarchy.
```suggestion
          <Prose textCenter>
            <h1>Change Paper</h1>
            <p>
              Load printer with <strong>legal-size paper</strong>.
            </p>
          </Prose>
```

Reply:
To be more useful to screen readers and other media, we should standardize on `<strong>` and `<em>` and avoid `<b>` and `<i>`.

Reply:
Ah nice, TIL about the difference between `strong` and `b`!